src/AccessibilityInsights/MainWindow.xaml.cs

```diff
@@ -198,6 +198,12 @@ private static void InitTelemetry()
             Logger.AddOrUpdateContextProperty(TelemetryProperty.AppSessionID, Guid.NewGuid().ToString());
             Logger.AddOrUpdateContextProperty(TelemetryProperty.SessionType, "Desktop");
             Logger.AddOrUpdateContextProperty(TelemetryProperty.ReleaseChannel, ConfigurationManager.GetDefaultInstance().AppConfig.ReleaseChannel.ToString());
+
+            string windowsVersion = NativeMethods.GetCurrentWindowsVersionForTelemetry();
+            if (!string.IsNullOrEmpty(windowsVersion))
```

Comment:
Should we log something if we can't get the version?

Reply:
We'll know if a problem occurs, so I don't think it's necessary

---
In reply to: [277715682](https://github.com/Microsoft/accessibility-insights-windows/pull/332#discussion_r277715682) [](ancestors = 277715682)